pre_commit.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"log"
+)
+
+const (
+	colorYellow = "\033[33m"
+	colorGreen  = "\033[32m"
+	colorReset  = "\033[0m"
+	textBold    = "\033[1m"
+	colorRed    = "\033[31m"
+
+	TypeFix            = "fix"
+	TypeDocumentation  = "docs"
+	TypeFeature        = "feat"
+	TypeRefactory      = "refactor"
+	TypeStyle          = "style"
+	TypeTest           = "test"
+	TypePerformance    = "perf"
+	TypeBreakingChange = "breaking change"
+	TypeBuild          = "build"
+	TypeCi             = "ci"
+	TypeExit           = "exit"
+)
+
+var (
+	deployTypeMap = map[string]string{
+		"1":  TypeFix,
+		"2":  TypeDocumentation,
+		"3":  TypeFeature,
+		"4":  TypeRefactory,
+		"5":  TypeStyle,
+		"6":  TypeTest,
+		"7":  TypePerformance,
+		"8":  TypeBreakingChange,
+		"9":  TypeBuild,
+		"10": TypeCi,
+		"0":  TypeExit}
+)
+
+func main() {
+	printRed("Welcome to Data Platform pre commit!")
+
+	commitType, err := getCommitType()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	commitMessage := getCommitMessage(0)
+
+	commitFullMessage := fmt.Sprintf("type: [%s], message: %s", commitType, commitMessage)
+
+	printGreenYellow("[Your commit message is] ", commitFullMessage)
+
+	commit(commitFullMessage)
+}
+
+func printGreenYellow(tag, text string) {
+	fmt.Println(textBold + colorGreen + tag + colorReset + colorYellow + text + colorReset)
+}
+
+func printRed(text string) {
+	fmt.Println(colorRed + text + colorReset)
+}
+
+func printGreen(text string) {
+	fmt.Println(colorRed + text + colorReset)
+}
+
+func getCommitType() (string, error) {
+	printGreenYellow("[COMMIT TYPE]", ` Which one is your commit type? *insert the option number*
```

Comment:
@esequielvirtuoso I think you should use white instead. it looks like a warning message